ts/main.ts

```diff
@@ -0,0 +1,65 @@
+const $apiKey = 'tjsESTV0ZCq5TAudXkKvHgf2h8rRIqIkhHziRF5i';
+const $stateSelect = document.querySelector('#states') as HTMLSelectElement;
+const $parksContainer = document.createElement('div');
+const $overlay = document.querySelector('.overlay') as HTMLElement;
+document.body.appendChild($parksContainer);
```

Comment:
Why create this element in JavaScript if you are going to immediately append it? Why not create it in your HTML?